static/js/publisher/release/actions/availableRevisionsSelect.js

```diff
@@ -6,3 +14,34 @@ export function setAvailableRevisionsSelect(value) {
     payload: { value }
   };
 }
+
+export function selectAvailableRevisions(value) {
+  return (dispatch, getState) => {
+    dispatch(setAvailableRevisionsSelect(value));
+    dispatch(clearSelectedRevisions());
+
+    const state = getState();
+
+    // for each architecture
+    const archs = getArchitectures(getState());
```

Comment:
pass `state` through rather then the `getState` func?

Reply:
Ah, missed that one.